pmm/pmm2-server-release.groovy

```diff
@@ -217,6 +221,38 @@ pipeline {
                 """
             }
         }
+        stage('Publish OVF image') {
+            agent {
+                label 'virtualbox'
+            }
+            steps {
+                withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+                    sh """
+                        aws s3 cp --only-show-errors s3://percona-vm/\${OVF_VERSION} pmm2-server-\${VERSION}.ova
+                    """
+                }
+                sh """
+                    ssh -i ~/.ssh/id_rsa_downloads -p 2222 jenkins@jenkins-deploy.jenkins-deploy.web.r.int.percona.com "mkdir -p /data/downloads/pmm2/\${VERSION}/ova" || true
+                    sha256sum pmm2-server-\${VERSION}.ova > pmm2-server-\${VERSION}.sha256sum
```

Comment:
@AlekSi  Do we need to use "pmm2-server-${VERSION}" or "pmm-server-${VERSION}" as a name for ovf image ?

Reply:
For Docker we use (at https://www.percona.com/downloads/pmm2/) `pmm-server-2.1.0`, so I guess we should do the same for OVF.
@rnovikovP ?

Reply:
Yes. Pmm2-client was the only one place where we changed the name